arcgis-ios-sdk-samples/Layers/ArcGIS vector tiled layer (custom style)/VectorTileCustomStyleViewController.swift

```diff
@@ -17,40 +17,89 @@ import UIKit
 import ArcGIS
 
 class VectorTileCustomStyleViewController: UIViewController, VectorStylesVCDelegate {
-    @IBOutlet var mapView: AGSMapView!
-    
-    private var itemIDs = ["1349bfa0ed08485d8a92c442a3850b06",
+    @IBOutlet var mapView: AGSMapView! {
+        didSet {
+            mapView.map = AGSMap()
+        }
+    }
+    @IBOutlet var changeStyleBBI: UIBarButtonItem!
```

Comment:
Please [avoid abbreviations](https://swift.org/documentation/api-design-guidelines/#use-terminology-well) except for highly recognized standard ones, like URL:

![image](https://user-images.githubusercontent.com/2257493/96331742-a47d5680-1014-11eb-8ed6-43f65c940e3e.png)

For variables whose type is `UIBarButtonItem`, I recommend following Apple's [convention](https://developer.apple.com/documentation/uikit/uiviewcontroller/1621471-editbuttonitem) of using `BarItem` as a suffix.